src/App.tsx

```diff
@@ -1,24 +1,45 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useContext, useEffect, useState } from 'react';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoNotification } from './components/TodoNotification';
+import { TodoContext } from './context/TodoContext';
+import { getTodos } from './api/todos';
+import { ErrorContext } from './context/ErrorContext';
+import { USER_ID } from './utils/variables';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const { todos, setTodos } = useContext(TodoContext);
+  const { setErrorMessage } = useContext(ErrorContext);
```

Comment:
Create hooks for contexts so you don't need to pass context every time, for example:
```javascript
const useTodo = () => useContext(TodoContext);
```